<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settings</class>
 <widget class="QDialog" name="settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QListWidget" name="telephonesList">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <widget class="QWidget" name="accountInfo">
        <attribute name="title">
         <string>Account Information</string>
        </attribute>
        <widget class="QWidget" name="formLayoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>70</y>
           <width>361</width>
           <height>35</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Description</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="descriptionEdit"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="formLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>120</y>
           <width>361</width>
           <height>35</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Full Name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="nameEdit"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="formLayoutWidget_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>170</y>
           <width>361</width>
           <height>35</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Domain</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="domainEdit"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="formLayoutWidget_4">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>220</y>
           <width>361</width>
           <height>35</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>User Name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="usernameEdit"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="formLayoutWidget_5">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>270</y>
           <width>361</width>
           <height>35</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Password</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="passwordEdit"/>
          </item>
         </layout>
        </widget>
        <widget class="QCheckBox" name="activeCheckbox">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>94</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Activate</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="advanced">
        <attribute name="title">
         <string>Advanced</string>
        </attribute>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="addAccount">
     <property name="text">
      <string>Adicionar</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="removeAccount">
     <property name="text">
      <string>Remover</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>settings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>487</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>settings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>292</x>
     <y>493</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
